lib/util.js

```diff
@@ -444,8 +444,8 @@ exports.jitteredSleepTime = function (numofRetries, currentSleepTime, totalTimeo
  * @returns {Boolean} return a random number between two numbers.
  */
 function chooseRandom (firstNumber, secondNumber) {
-  let bigNumber = Math.max(firstNumber, secondNumber);
-  let smallNumber = Math.min(firstNumber, secondNumber);
+  const bigNumber = Math.max(firstNumber, secondNumber);
+  const smallNumber = Math.min(firstNumber, secondNumber);
```

Comment:
It doesn't really matter which one is bigger. 
Math.random() * (firstNumber - secondNumber)) + secondNumber
would always return what we need.

Reply:
Fixed